plugins/inputs/couchbase/README.md

```diff
@@ -22,7 +22,7 @@
 ### couchbase_node
 
 Tags:
-- cluster: whatever you called it in `servers` in the configuration, e.g.: `http://couchbase-0.example.com/`
+- cluster: sanitized and scheme less string from `servers` configuration field e.g.: `http://user:password@couchbase-0.example.com:8091/endpoint` -> `couchbase-0.example.com:8091/endpoint`
```

Comment:
I think it is better to use scheme less name for displaying and creating dashboard. In other hand, it may cause problems when there will be same endpoint using different protocols (scheme). If you wish I can also add scheme into this string, just let me know.

Thanks!